src/SQLQueryStress/Program.cs

```diff
@@ -14,47 +15,72 @@ internal static class Program
         /// </summary>
         [STAThread]
         private static void Main(string[] args)
+        {
+            var options = new CommandLineOptions();
+            var parserResult = Parser.Default.ParseArguments<CommandLineOptions>(args);
+
+            parserResult
+                .WithParsed(options => Run(options))
+                .WithNotParsed(errors => DisplayHelp(parserResult, errors));
+        }
+
+        private static void Run(CommandLineOptions options)
         {
             AppDomain.CurrentDomain.AssemblyResolve += OnResolveAssembly;
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
 
+            using var form1 = new Form1(options)
+            {
+                StartPosition = FormStartPosition.CenterScreen
+            };
+            Application.Run(form1);
+        }
+
+        private static void DisplayHelp<T>(ParserResult<T> result, IEnumerable<Error> errors)
+        {
+            NativeMethods.AttachParentConsole();
 
-            var options = new CommandLineOptions();
-            ICommandLineParser parser = new CommandLineParser();
-            var writer = new StringWriter();
-            parser.ParseArguments(args, options, writer);
-            
-            if (writer.GetStringBuilder().Length > 0)
+            if (errors.IsVersion())
             {
-                MessageBox.Show(writer.GetStringBuilder().ToString());
+                Console.WriteLine(HelpText.AutoBuild(result));
             }
-            else
+            else if (errors.IsHelp())
             {
-                var f = new Form1(options)
+                var helpText = HelpText.AutoBuild(result, help =>
                 {
-                    StartPosition = FormStartPosition.CenterScreen
-                };
-                Application.Run(f);
+                    help.AdditionalNewLineAfterOption = true;

```

Comment:
Can we get the version and copyright information back also?

Reply:
They were also moved to HelpText.AutoBuild related to CommandLineParser changes.

Reply:
I dont see it, but maybe you can share a screen  shot of the help screen?

Reply:
![image](https://user-images.githubusercontent.com/46247241/92381469-96820f00-f113-11ea-8857-18570cf1bf15.png)